### ISSUE TITLE
Merge with other embedded system project?

### ISSUE THREAD

**Issue by fkromer** [embedded-boston/awesome-embedded#13]
I started some project about embedded systems learning resources some while ago and [hosted it with readthedocs.org](http://awesome-embedded-systems.org/). Seems like the projects are overlapping to a great extend (my project is missing frameworks but lists much more books, MOOCs, etc.). Would you be interested to merge?

**Comment by cwoodall** [comment]
I would be interested in merging. I wanted to follow a formatting that is used by other awesome lists rather than a website based list (so markdown, and in the format described by https://github.com/sindresorhus/awesome).

I am up for a discussion on this front, the content on your list is great and I would like to integrate the lists some how.

**Comment by fkromer** [comment]
I do not care about project or content ownership. I care about avoiding dublication and ease of maintenance. I like the awesome-list functionality to check for broken URLs. I did not know it. I am fine with integrating my content into your project. For referencing books I prefer some bibliography style (plus ISBN) instead of plain URL hyperlinking to some book store website. What formating can be used in your project for something like that?

**Comment by cwoodall** [comment]
I have a few proposals. Not 100% on which of them is best.

We could enforce a bibliography style of some sort on merges. Maybe just keep the list format of traditional awesome-lists rather than using a table?

- [Authors. Book Title, Edition. Publisher, Year.](link if one is provided) - Description

**Comment by fkromer** [comment]
The traditional list format should be ok. I would consider a hyperlink to a books table of content helpful. Many book publishers provide that on the book landing page e.g. [Building Embedded Linux Systems](http://shop.oreilly.com/product/9780596529680.do).

**Comment by fkromer** [comment]
@cwoodall I would like to pass my repo name **awesome-embedded-systems** on to you. This name suits the list content better than **awesome-embedded**, right? Let me know if you are interested and I will change my repo name and merge the content...

**Comment by cwoodall** [comment]
@fkromer I agree.

Would you mind if I made it so you were able to manage this repo along with me?

**Comment by fkromer** [comment]
@cwoodall yeah, why not. I would have put some time into this topic either way.

**Comment by fkromer** [comment]
@cwoodall you can take over the repo name now...
